README.md

```diff
@@ -15,7 +15,9 @@ git clone https://github.com/pypsa-meets-earth/pypsa-earth-lit.git
 
 # depending on what you need. Recommended order.
 make -f ./pypsa-earth-lit/Makefile get_data
```

Comment:
I think this function will currently install `pypsa-earth` results in this way:
```
tauritron-project/pypsa-earth
tauritron-project/pypsa-earth-lit
```
However, as `pypsa-earth` is hard-coded in later script,
it currently runs only when the pypsa-earth data is in 
`pypsa-earth-lit/pypsa-earth`

Suggestion. Don't hard code `pypsa-earth` across scripts.
Keep if possible the original structure:
```
tauritron-project/pypsa-earth
tauritron-project/pypsa-earth-lit
```
and test if it works.

Reply:
I have changed the pypsa earth path and tested it with the readme file it's working fine now.